cubequery/api_server.py

```diff
@@ -237,25 +206,5 @@ def normalise_task_info(info):
 
     return result
 
-
-def validate_app_key():

```

Comment:
instead of this how are we checking that this is only available from the proxy? I assume we will be needing a change in the helm charts so cube query isn't accessable outside the cluster?